src/dodal/beamlines/i19_2.py

```diff
@@ -75,3 +76,11 @@ def synchrotron() -> Synchrotron:
     If this is called when already instantiated in i19-2, it will return the existing object.
     """
     return Synchrotron()
+
+
+@device_factory()
+def backlight() -> BacklightPosition:
+    """Get the i19-2 backlight device, instantiate it if it hasn't already been.
+    If this is called when already instantiated in i19-2, it will return the existing object.
+    """
+    return BacklightPosition(prefix=(PREFIX.beamline_prefix))
```

Comment:
```suggestion
    return BacklightPosition(prefix=f"{PREFIX.beamline_prefix}-EA-IOC-12:")
```
Nit: As a general rule, we tend to pass the full prefix here, instead of just the beamline bit, unless there's reasons for it like differently named PVs in the device